homeassistant/util/yaml.py

```diff
@@ -6,6 +6,7 @@
 import glob
 import yaml
 
+import homeassistant.util.secrets as secrets
```

Comment:
Just an idea, but exposing as a homeassistant util, component designers can force secret usage.

e.g. if http component do not get `api_password` in config file, it queries `get_secret` directly. Or even force sensitive things to be secret and not use config files at all, encouraging good behaviour

We can reduce to a single function, but then we'll try loading keyring in yaml.py, if thats ok?